ui/admin/tests/acceptance/storage-buckets/selectors.js

```diff
@@ -30,3 +31,8 @@ export const FIELD_WORKER_FILTER = '[name=worker_filter]';
 export const FIELD_WORKER_FILTER_ERROR = '[data-test-worker-filter-error]';
 export const TOAST = '[role=alert] div';
 export const TOAST_SUCCESSFULL_VALUE = 'Saved successfully.';
+
+export const TABLE_FIRST_ROW_ACTION_DROPDOWN =
+  'tbody tr:first-child td:last-child button';
+export const TABLE_FIRST_ROW_ACTION_FIRST_ITEM =
+  'tbody tr:first-child td:last-child ul li:first-child a';
```

Comment:
These selector ultimately could fall under "common selectors" but when it comes to tables and using selectors to interact or assert on certain elements, we should keep it close to the resource we are working with. Happy to discuss this more and/or change to a common selector.

Reply:
Agreed, this granularity of selector with pseudo-class is best to keep it close to its domain.
Something like: `tbody button.deleteElement` can fit better as common selector. Since it's using very general selector + a class.